### ISSUE TITLE
libX11 xlibi18n unused?

### ISSUE THREAD

**Issue by uli42** [ArcticaProject/nx-libs#264]
The following stuff is currently not used during build. Either fix that or remove them. Check, what upstream does.

```
./nx-X11/lib/X11/xlibi18n:                                    
-rw-rw-r-- 1 uli uli  234 Okt 29 21:27 Imakefile
-rw-rw-r-- 1 uli uli 3132 Okt 29 21:27 Xi18nLib.conf

./nx-X11/lib/X11/xlibi18n/im:
-rw-rw-r-- 1 uli uli 296 Okt 29 21:27 Imakefile

./nx-X11/lib/X11/xlibi18n/im/ximcp:
-rw-rw-r-- 1 uli uli 1767 Okt 29 21:27 Imakefile
-rw-rw-r-- 1 uli uli   41 Okt 29 21:27 Ximcp.mapfile

./nx-X11/lib/X11/xlibi18n/lc:
-rw-rw-r-- 1 uli uli 303 Okt 29 21:27 Imakefile

./nx-X11/lib/X11/xlibi18n/lc/def:
-rw-rw-r-- 1 uli uli 256 Okt 29 21:27 Imakefile
-rw-rw-r-- 1 uli uli  48 Okt 29 21:27 Xlc.mapfile

./nx-X11/lib/X11/xlibi18n/lc/gen:
-rw-rw-r-- 1 uli uli 260 Okt 29 21:27 Imakefile
-rw-rw-r-- 1 uli uli  48 Okt 29 21:27 Xlc.mapfile

./nx-X11/lib/X11/xlibi18n/lc/Utf8:
-rw-rw-r-- 1 uli uli 265 Okt 29 21:27 Imakefile

./nx-X11/lib/X11/xlibi18n/om:
-rw-rw-r-- 1 uli uli 298 Okt 29 21:27 Imakefile

./nx-X11/lib/X11/xlibi18n/om/generic:
-rw-rw-r-- 1 uli uli 577 Okt 29 21:27 Imakefile
-rw-rw-r-- 1 uli uli  48 Okt 29 21:27 Xom.mapfile 
```

**Comment by uli42** [comment]
This posting by Orion Popalski indicates we can drop them, I'd say: https://lists.fedoraproject.org/pipermail/scm-commits/Week-of-Mon-20131007/1126343.html

**Comment by sunweaver** [comment]
uli42: Very cool! Will you look into dropping that xlibi18n code completely, then? Or shall I?

**Comment by uli42** [comment]
I can do it this evening I think but go on if you like. Just let me know.

Vom Smartphone gesendet.

----- Ursprüngliche Nachricht -----
Von: "Mike Gabriel" notifications@github.com
Gesendet: ‎03.‎11.‎2016 10:51
An: "ArcticaProject/nx-libs" nx-libs@noreply.github.com
Cc: "Ulrich Sibiller" uli42@gmx.de; "Assign" assign@noreply.github.com
Betreff: Re: [ArcticaProject/nx-libs] libX11 xlibi18n unused? (#264)

uli42: Very cool! Will you look into dropping that xlibi18n code completely, then? Or shall I?

**Comment by sunweaver** [comment]
Fix available as PR #279.
